lib/onebox/engine/bandcamp_onebox.rb

```diff
@@ -9,13 +9,12 @@ class BandCampOnebox
 
       def placeholder_html
         og = get_opengraph
-        escaped_src = ::Onebox::Helpers.normalize_url_for_output(og[:image])
-        "<img src='#{escaped_src}' height='#{og[:video_height]}' #{Helpers.title_attr(og)}>"
+        "<img src='#{og.image}' height='#{og.video_height}' #{og.title_attr}>"
       end
 
       def to_html
         og = get_opengraph
-        src = og[:video_secure_url] || og[:video]
+        src = og.video_secure_url || og.video
         escaped_src = ::Onebox::Helpers.normalize_url_for_output(src)
```

Comment:
I think both `video_secure_url` and `video` are already normalized. Doing it one more time might cause problems. 🤔